Traceback (most recent call last):
  File "/Users/samuel.oranyeli/mambaforge/envs/blogger/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/samuel.oranyeli/mambaforge/envs/blogger/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/samuel.oranyeli/mambaforge/envs/blogger/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/samuel.oranyeli/mambaforge/envs/blogger/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/samuel.oranyeli/mambaforge/envs/blogger/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/samuel.oranyeli/mambaforge/envs/blogger/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/samuel.oranyeli/mambaforge/envs/blogger/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/samuel.oranyeli/mambaforge/envs/blogger/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
frame = xlsx_cells(filename, sheetnames="annotations", fill=True).astype(
    {"row": np.int8, "column": np.int8}
)
frame
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[9], line 1[0m
[0;32m----> 1[0m frame [38;5;241m=[39m [43mxlsx_cells[49m(filename, sheetnames[38;5;241m=[39m[38;5;124m"[39m[38;5;124mannotations[39m[38;5;124m"[39m, fill[38;5;241m=[39m[38;5;28;01mTrue[39;00m)[38;5;241m.[39mastype(
[1;32m      2[0m     {[38;5;124m"[39m[38;5;124mrow[39m[38;5;124m"[39m: np[38;5;241m.[39mint8, [38;5;124m"[39m[38;5;124mcolumn[39m[38;5;124m"[39m: np[38;5;241m.[39mint8}
[1;32m      3[0m )
[1;32m      4[0m frame

[0;31mNameError[0m: name 'xlsx_cells' is not defined
NameError: name 'xlsx_cells' is not defined

